test/parallel/test-http-incoming-pipelined-socket-destroy.js

```diff
@@ -21,11 +21,12 @@
 
 'use strict';
 const common = require('../common');
+const Countdown = require('../common/countdown');
 
 const http = require('http');
 const net = require('net');
-
```

Comment:
Hi @idandagan1 — one little change before this lands, could you re-insert this line break between the requires and the rest of the code?

Reply:
Done.